spec/index.bs

```diff
@@ -1178,6 +1179,30 @@ On setting the {{HTMLElement/innerText}}, {{Node/textContent}} and {{HTMLScriptE
 1. Set {{HTMLScriptElement/[[ScriptText]]}} internal slot value to the stringified attribute value.
 1. Perform the usual attribute setter steps.
 
+#### Setting slot values from parser #### {#setting-slot-values-from-parser}
+
+This document modifies the HTML parser to set the {{HTMLScriptElement/[[ScriptText]]}} internal slot value when the script is created.
+
+Modify the [=The text insertion mode=] algorithm as follows:
+
+<dl class="switch">
+    <dt id="scriptEndTag">An end tag whose tag name is "script"</dt>
+    <dd>
+        <p>...</p>
+
+        <ins>Set <var>script</var>'s <a href="#slots-with-trusted-values">[[ScriptText]]</a> internal slot value to its
```

Comment:
We probably should also check that the internal slot isn't already set, if it is and we reach here (not even sure how it would happen?) we should probably leave it alone or maybe even reset to an empty string?

This way it ensures in the event of shenanigans that the spec plays it safe and assumes untrusted

Reply:
For now I'm going to leave this out because that's partly caused by the API stuff which I'm defering to a follow up.